site/app/models/gradeable/Gradeable.php

```diff
@@ -90,8 +90,8 @@ class Gradeable extends AbstractModel {
     protected $instructions_url = "";
     /** @property @var int The type of gradeable */
     protected $type = GradeableType::ELECTRONIC_FILE;
-    /** @property @var bool If the gradeable should be graded per registration section (true) or rotating sections(false) */
-    protected $grade_by_registration = true;
+    /** @property @var int If the gradeable should be graded by all access (2) by registration section (1) or rotating sections (0) */
+    protected $grader_assignment_method = 1;
```

Comment:
can you make this an enumeration instead of a plain integer (like `Gradeable::GRADE_BY_REGISTRATION`, etc.)?   This way the values will be self-documenting and when they're used scattered throughout the code they won't look like "magic numbers".

Reply:
I made this change as suggested.  I did it within the Gradeable file as only the constants were needed instead of in a new file (like GradeableType).  If this is bad practice or if there are any other suggestions please let me know!